test/index.spec.ts

```diff
@@ -471,6 +471,48 @@ describe("MockAxios", () => {
       expect(MockAxios.getReqByMatchUrl(new RegExp('right'))).toStrictEqual(firstReq);
     });
 
+    let firstReq
```

Comment:
Can we reduce the scope of these two variables by moving them into the `describe()` block below? Otherwise they are getting shadowed by variables with the same name in the test cases above, which is a bit ugly and also noted by tslint (`Shadowed name: 'firstReq'`)

Reply:
Sure!

Reply:
Done!